initrd/bin/gui-init

```diff
@@ -547,7 +547,7 @@ prompt_totp_mismatch()
 {
   TRACE_FUNC
   if (whiptail_warning --title "TOTP/HOTP code mismatched" \
-    --yesno "TOTP/HOTP code mismatches could indicate TPM tampering or clock drift.\n\nThe current UTC time is: $(date "+%Y-%m-%d %H:%M:%S %Z")\nIf this is incorrect, set the correct time and check TOTP/HOTP again.\n\nDo you want to change the time?" 0 80) then
+    --yesno "TOTP/HOTP code mismatches could indicate TPM tampering or clock drift.\n\nThe current UTC time is: $(date "+%Y-%m-%d %H:%M:%S")\nIf this is incorrect, set the correct time and check TOTP/HOTP again.\n\nDo you want to change the time?" 0 80) then
```

Comment:
This is from whiptail menu, not console.
This might be redundant and most probably more relevant in change-time directly @JonathonHall-Purism, since this is where time change will occur anyway? Or duplicate, I don't really mind. Its just that change-time is called from whiptail and from another menu which doesn't show time right now, and call clear in the script. So would be better under change-time in my opinion.

Reply:
https://github.com/linuxboot/heads/pull/1748/files#r1705908889